litestar/_openapi/schema_generation/schema.py

```diff
@@ -116,6 +116,7 @@
     MutableMapping: Schema(type=OpenAPIType.OBJECT),
     MutableSequence: Schema(type=OpenAPIType.ARRAY),
     None: Schema(type=OpenAPIType.NULL),
+    type(None): Schema(type=OpenAPIType.NULL),
```

Comment:
Nit, but we do have an alias for this:

`from litestar.types.builtin_types import NoneType`